### PR TITLE
update program

### PR DESCRIPTION
**Pull request by smiile8888**
- update program to date
- style sponsor section

### CODE REVIEW

File: css/dlai5_styles.css [hunk 1]
```diff
@@ -356,3 +356,9 @@ tr:nth-child(even),
   border-bottom: 1px solid #c1c1c1;
   padding-bottom: 12px;
 }
+
+.flex {
+  display: flex;
+  justify-items: center;
+  align-items: center;
+}
```
Comment (JaturongKongmanee) [inline css/dlai5_styles.css:364]: What are these for?

Comment (JaturongKongmanee) [reply css/dlai5_styles.css:364]: I mean what does the "flex" do?

Comment (smiile8888) [reply css/dlai5_styles.css:364]: It makes elements inside aligned center, vertically and horizontally

**Review by JaturongKongmanee — APPROVED** [review]
Thanks for clarifying. 
LGTM!

**Pull request merged**